### PR TITLE
#164069266 Setup Travis CI for unit tests 

### PR DESCRIPTION
**Pull request by Stanley-Okwii**
#### **What does this PR do?**
- enables Travis CI to run unit tests

#### **Description of Task to be completed?**
- enables Travis CI to run unit tests

#### **How should this be manually tested?**
- Click on [here](https://travis-ci.org/andela/ah-groot-frontend/branches) to view successful builds on Travis.

#### **Screenshots**
Screenshot 2019-04-01 at 20 44 18

#### **What are the relevant pivotal tracker stories?**
[ #164069266](https://www.pivotaltracker.com/story/show/164069266)

### CODE REVIEW

**Review by NLSanyu — APPROVED** [review]
LGTM

**Review by nyabongoedgar — APPROVED** [review]

**Review by akram256 — APPROVED** [review]
LGTM

File: src/components/App.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ import {
   Route,
```
Comment (huxaiphaer) [inline src/components/App.js]: Why is the App component in the **component** folder? we agreed that all stateful components should be in the container folder. Secondly, your task is only about Travis, why are you editing other areas, please stay in your boundary.

File: src/index.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { Provider } from 'react-redux';
-import App from './containers/App';
+import App from './components/App';
```
Comment (huxaiphaer) [inline src/index.js:4]: Same applies here. why did you change this ?

File: src/components/App.js (hunk 1 above)
Comment (Stanley-Okwii) [reply src/components/App.js]: > Why is the App component in the **component** folder? we agreed that all stateful components should be in the container folder. Secondly, your task is only about Travis, why are you editing other areas, please stay in your boundary.

@huxaiphaer , I thought because **App**  component didn't initialize state and that it did not have any redux dispatch actions that it deserved to be in the components folder. Am fixing this now. Will let you know when it's done.

File: src/index.js (hunk 2 above)
Comment (Stanley-Okwii) [reply src/index.js:4]: Working on it as well.

**Review by walimike — APPROVED** [review]

**Comment by huxaiphaer** [comment]
LGTM
